### ISSUE TITLE
Blender animation support 

### ISSUE THREAD

**Issue by alexderpyfox** [Neos-Metaverse/NeosPublic#3465]
### Is your feature request related to a problem? Please describe.

It doesn't import avatars with blender animation 

### Relevant issues

Animation

### Describe the solution you'd like

Blender animation so you can animate the avatar with it 

### Describe alternatives you've considered

I don't know anymore i tried everything

**Comment by Electronus** [comment]
There is some complexity here, you can't animate bones that the IK uses, and when IK sets up it removes all animations. you can import a non-IK version and pull the animations off of that however and if they dont need the same bones as the IK it will work just fine.

**Comment by alexderpyfox** [comment]
Yes and the blender animation isn't importable so when I try to import an avatar that got blender animation it imports it as an avatar instead so then the animation doesn't get imported

**Comment by Electronus** [comment]
Yeah then you import it again and in the advanced options tell it to not set up the IK, then you can move it over! would be nice if you didn't have too, or im missing something.
